packages/webpack-config/src/webpack/base/tasks/jsLoading/compileES/module.js

```diff
@@ -1,4 +1,4 @@
-const path = require('path');
```

Comment:
I don’t think anything has changed in this file, accept formatting.

Reply:
i'll postpone this revert since we'll introduce the eslint config shortly and reformat all files accordingly